source/3ds/gui_hard.c

```diff
@@ -1969,7 +2327,7 @@ int guiGetInput(bool ingame) {
                 return ydist <= 0 ? VB_RPAD_U : VB_RPAD_D;
             }
         }
-    } else if (hidKeysDown() & KEY_TOUCH && (touch_pos.px > 32 || touch_pos.py > 32)) {
```

Comment:
So this causes a frustrating bug where if the backlight is off, and you happen to touch where the backlight icon was, the screen won't turn on, which is very confusing, so I removed it, let me know if this behavior was intentional for some reason...

Reply:
Good catch. That's been there since the original commit. There was a conflict with the touchscreen settings menu at some point, but I checked and this is fine here.